packages/cosmos_node_client/analysis_options.yaml

```diff
@@ -0,0 +1,228 @@
+include: package:lint/analysis_options.yaml
```

Comment:
let's use `cosmos_lint` package here instead of specifying the rules again

Reply:
Yeah I'm going to do that just needed the previous PR to be merged.